lib/src/room.dart

```diff
@@ -393,6 +395,16 @@ class Room {
     return lastEvent;
   }
 
+  void setEphemeral(BasicRoomEvent ephemeral) {
+    ephemerals[ephemeral.type] = ephemeral;
```

Comment:
Do you think this could remove a typing indicator incorrectly in a room when there are multiple people typing at the same time? Should we scope the timer to the sender of the ephemeral as well?

Reply:
Guess that won't happen because the timer would be reset with the latest ephemeral update.

Reply:
But that would mean:

A starts typing at 00s
B starts typing at 29s
The timer is reset now, so the ephemeral wouldn't clear A's typing status and both A & B would be shown typing.

Maybe we should scope to the sender id?

Reply:
Yes that would be better in theory. However, it would be much more complex for a rare edge case 🤔 we would need to change the way how we handle ephemerals completely in the Room class

Reply:
So we would lose the "We just store the ephemeral as it is" approach and go to a "handle typing indicators special". I don't wanna sound lazy but I would personally prefer a less complex solution with such a rare edge case. What do you think?

Reply:
Hmm, sounds good to me. But do you mind writing an issue for this edge case somewhere so we can tackle it in the future with a different approach maybe?

I can also do that if you want. Please feel free to resolve this conversation :)

Reply:
Would be great if you can do it <3